sota-implementations/multiagent/maddpg_iddpg.py

```diff
@@ -200,7 +203,7 @@ def train(cfg: "DictConfig"):  # noqa: F821
                 optim.zero_grad()
                 target_net_updater.step()
 
-        policy_explore.step(frames=current_frames)  # Update exploration annealing
+        policy_explore[1].step(frames=current_frames)  # Update exploration annealing
```

Comment:
in the example I gave above, the `update_exploration` or `step_exploration` should be turned into a class to allow us to pass the `current_frames`